data_generator.py

```diff
@@ -29,6 +29,16 @@
 np.set_printoptions(suppress=True)
 # -----------------------------------------------------------
 
+# -------------------SENSORS LIST INDEXES--------------------
```

Comment:
I'm not sure if I like this practice of hardcoding the indexes. Seems arbitrary, especially because in `init_sensors()` you are hardcoding their placement anyways (`[gyro, thermistor, baro_pressure_sensor, accelerometer, magnetometer]`)